src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--rating-4 .stars__star:nth-child(-n + 4) {
```

Comment:
The selector ".stars--rating-4 .stars__star:nth-child(-n + 4)" is very specific. Consider using a more general approach if possible, to make the code more maintainable. This is a suggestion based on best practices for CSS specificity.